### PR TITLE
Remove JS A/B-verification code

### PR DESCRIPTION
**Pull request by Swatinem**
We were still sampling a very low number of events here, but noone has been looking at any differences since quite a while

### CODE REVIEW

File: src/sentry/lang/javascript/plugin.py [hunk 1]
```diff
@@ -44,9 +44,7 @@ def get_event_preprocessors(self, data, **kwargs):
         return []
 
     def get_stacktrace_processors(self, data, stacktrace_infos, platforms, **kwargs):
-        if data.get("processed_by_symbolicator", False) and not data.get(
-            "symbolicator_stacktraces"
-        ):
+        if data.get("processed_by_symbolicator", False):
```
Comment (loewenheim) [inline src/sentry/lang/javascript/plugin.py:47]: Do we even still need `processed_by_symbolicator` at this point?

**Review by loewenheim — APPROVED** [review]

File: src/sentry/lang/javascript/plugin.py (hunk 1 above)
Comment (kamilogorek) [reply src/sentry/lang/javascript/plugin.py:47]: I'd drop it. We already have all other meta fields that only symbolicator produces

Comment (Swatinem) [reply src/sentry/lang/javascript/plugin.py:47]: I  will double-check, as we already had bugs related to putting a mixed-stacktrace event through the two different symbolicators.

Comment (Swatinem) [reply src/sentry/lang/javascript/plugin.py:47]: Yes, I believe we should keep the current logic as is, and revisit this in https://github.com/getsentry/sentry/pull/49675.
That PR already has merge conflicts, and will probably also conflict with this one here.

But once we have that done, we should be able to simplify this logic a bit.

**Pull request merged**